nornir/plugins/tasks/connections/paramiko_connection.py

```diff
@@ -35,6 +35,11 @@ def paramiko_connection(task=None):
     if "proxycommand" in user_config:
         parameters["sock"] = paramiko.ProxyCommand(user_config["proxycommand"])
 
+    task.host["ssh_forwardagent"] = False
```

Comment:
can you turn into a private attribute (i.e. `task.host._ssh_forward_agent`), please? I don't want to write into the `Host` inventory data. To avoid problems make sure you initialize the same attribute in `Host.__init__()` as `None`.